### ISSUE TITLE
QRCode credential

### ISSUE THREAD

**Issue by Maxhy** [leosac/leosac#139]

**Comment by omnisci3nce** [comment]
I like it! Would be useful for guest passes which is perhaps what you mean by #140

**Comment by Maxhy** [comment]
Yes that would be the main purpose of it indeed. It will take a while to get there as credential refactoring is required first but it is definitely a goal.
